docs/variants/dell_optiplex/recovery.md

```diff
@@ -1,5 +1,205 @@
+
+<center>
+![](../../images/dell_optiplex_9010_spi_pin1.jpg)
+</center>
+
+## Step 3: Connect SOIC-8 Pomona clip between RTE and target
+
+Connect SOIC-8 Pomona according to [MX25L3206E datasheet](https://www.macronix.com/Lists/Datasheet/Attachments/8616/MX25L3206E,%203V,%2032Mb,%20v1.5.pdf).
+
+<center>
+![](../../images/mx25l3206e_pinout.png)
+</center>
+
+<center>
+ ![Pomona SOIC clip](../../images/pomona_clip.jpg)
+</center>
+
+<center>
+
+ | RTE J7                                 | Pomona SOIC clip  |
+ |:--------------------------------------:|:-----------------:|
+ | CS                                     | pin 1 (upside)    |
+ | MISO                                   | pin 2 (upside)    |
+ | GND                                    | pin 4 (upside)    |
+ | VCC                                    | pin 5 (downside)  |
+ | SCLK                                   | pin 7 (downside)  |
+ | MOSI                                   | pin 8 (downside)  |
+
+</center>
+
+Numbers 1-4 have to be on one side and numbers 5-8 have to be on the other side
+of the clip.
+
+<center>
+![Clip up](../../images/clip_upside.jpg)
+</center>
+
+<center>
+![Clip down](../../images/clip_downside.jpg)
+</center>
+
+Clip on the `SPI_1` chip. Match pin 1 (`CS`) of the Pomona clip with the first
+pin of `SPI_1` chip, marked with a small dot engraved on the chip.
+
+![Clip connected](../../images/clip_connected.jpg)
+
+## Step 4: Connect RTE
+
+<center>
+![](../../images/rte-boot.jpg)
+</center>
+
+1. Connect J2 Orange Pi Zero system debug output
+2. Power the board and confirm it boots
+3. **Please note** typical convention of USB-UART converter colors is as follows
+    - black - GND
+    - red - +5V
+    - green - TX
+    - white - RX
+4. Connect terminal to RTE and read OS version:
+
+    ```shell
+    sudo minicom -b 115200 -D /dev/ttyUSB0 -o -C /tmp/minicom.cap
+    ```
+
+    - `-b 115200` sets baudrate
+    - `-D /dev/ttyUSB0` points to USB-UART converter device, it can be
+      different if you already have some devices connected or you use different
+      operating system
+    - `-o` skip initialization
+    - `-C /tmp/minicom.cap` capture serial terminal output, if you will have
+      problems with exercises please post this file
+5. Login using following credentials:
+
+    ```shell
+    login: root
+    password: meta-rte
+    ```
+
+## Step 5: Prepare recovery binary
+
+* Get v0.2-rc3 binary
+
+```bash
+wget https://cloud.3mdeb.com/index.php/s/8WNEHEFcBGFRK23/download -O dasharo_dell_optiplex_9010_v0.2-rc3.rom
+```
+
+* It has 12MB, so it have to be split
+
+```bash
+split -b4M dasharo_dell_optiplex_9010_v0.2-rc3.rom
+```
+
+## Step 6: Flash 4MB (BIOS) part
+
+```bash
+echo 1 > /sys/class/gpio/gpio405/value
+```
+
+```bash
+echo 1 > /sys/class/gpio/gpio406/value
+```
+
+```bash
+echo 1 > /sys/class/gpio/gpio404/value
+```
+
+```bash
+flashrom -w xac -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -c "MX25L3205D/MX25L3208D"
```

Comment:
maybe a comment what `xac` is, I doubt everyone will understand it is the third file resulting from split

Reply:
Fixed https://github.com/Dasharo/docs/pull/139/commits/939d9a1569b1fe397c0f585ff79871535218aede